tests/functional/placementapi_controller_test.go

```diff
@@ -344,6 +344,9 @@ var _ = Describe("PlacementAPI controller", func() {
 			myCnf := cm.Data["my.cnf"]
 			Expect(myCnf).To(
 				ContainSubstring("[client]\nssl=0"))
+			configData := cm.Data["httpd.conf"]
+			Expect(configData).Should(
+				ContainSubstring("TimeOut 60"))
 		})
```

Comment:
Do we have a way to test the annotations on the Route? I guess not in functional env as the Route is only created by the openstack-operator but maybe in the kuttl env?

Reply:
At least we need to extend https://github.com/openstack-k8s-operators/openstack-operator/blob/main/tests/functional/ctlplane/openstackoperator_controller_test.go#L855 when we integrate this patch into the openstack-operator.